### DISCUSSION TITLE
Configure the sequence in which the sidebar groups are shown.

### DISCUSSION THREAD

**Discussion by iamlinkus** [payloadcms/payload#2447] [Feature Requests & Ideas]
Currently you can group the admin panel collections and globals into groups within the sidebar, which is awesome. Unfortunatelly, once you put globals into a group, it will always show those groups below the collection groups. It would be great to be able to config the sequence in which the groups are shown regardless of whether they're for globals or collections.

Here's an example:

Working with other cms'es we usually have a group for "Pages", which has all the pages listed and the user can change the content of each page. Each page having a different structure. Then we have another group for "Global settings", which includes stuff like the footer, header, default SEO, etc. And only below these groups we show the rest of the collection groups ("Media" group, which has icons, logos, etc or "Blog" group which has Posts, Comments).

Here's an example with the current payload functionality. The groups "Assets" and "Admin" are groups of collections. And "Pages" is a group for globals. All I need to do now is somehow move the "Pages" globals group to be above all the others.

![image](https://user-images.githubusercontent.com/16874184/229838122-c2d7c3bb-5c26-49d7-b7c8-ec95c3e98057.png)

One interessting thing that @AlessioGr pointed out in [Payload Discord](https://discord.com/channels/967097582721572934/1092477550397239347/1092823891728154755) is that if you don't put the globals in a custom group, then the default group "Globals" is above everything else, but then there's no possibility to change the group name and/or it doesn't work if you want to have more than one group of globals.

**Comment by christian-reichart** [comment]
+1 for this. 
What if we could write up a nested structure in the config? All Globals & Collections that aren't represented in the structure will be shown below with the current logic...?

So I think this could be thought bigger with allowing more layers (double nesting), adding Icons, etc...
